pkg/cmd/repo/edit/edit.go

```diff
@@ -132,16 +194,19 @@ func editRun(ctx context.Context, opts *EditOptions) error {
 
 	if len(opts.AddTopics) > 0 || len(opts.RemoveTopics) > 0 {
 		g.Go(func() error {
-			existingTopics, err := getTopics(ctx, opts.HTTPClient, repo)
-			if err != nil {
-				return err
+			var topics []string
+			if !opts.InteractiveMode {
+				var err error
+				topics, err = getTopics(ctx, opts.HTTPClient, repo)
+				if err != nil {
+					return err
+				}
 			}
-
 			oldTopics := set.NewStringSet()
-			oldTopics.AddValues(existingTopics)
+			oldTopics.AddValues(topics)
```

Comment:
Topics is no longer populated when in `InteractiveMode` which is why I had introduced `opts.Topics` so that when we retrieve them in `InteractiveMode` we wouldn't need to re-retrieve them here.